earth_enterprise/src/fusion/autoingest/AssetSerializer.h

```diff
@@ -5,6 +5,10 @@
 #include "common/khxml/khxml.h"
 #include "common/khxml/khdom.h"
 #include "autoingest/.idl/storage/AssetDefs.h"
+#include <khxml/khdom.h>
+using namespace khxml;
```

Comment:
It's generally a bad idea to include a `using namespace` declaration in a header file. We should remove that and just use `khxml::` as needed.